README.md

```diff
@@ -12,6 +12,10 @@ The Graph Explorer is written in [TypeScript](https://www.typescriptlang.org/) a
 * `bower install`
 * `npm start` starts the TypeScript compiler in watch mode and the local server. It should open your browser automatically with the Graph Explorer at [http://localhost:3000/](http://localhost:3000).
 
+#### Enabling authentication with your own credentials
+* You'll need to register an app on [apps.dev.microsoft.com](https://apps.dev.microsoft.com) to configure the login page for your local Graph Explorer.  Under `Platforms` click `Add Platform` and select Web.  `Allow Implicit Flow` should be checked and set `http://localhost:3000` as the redirect URL.  You don't need a client secret since the explorer is a single page application.
+* Rename `secrets.sample.js` to `secrets.js` in the project root and insert your client ID.
```

Comment:
I just realized that it has to happen that way since the gitignore would make it so the secrets* file wouldn't be cloned.